src/scripts/services/user-service.ts

```diff
@@ -0,0 +1,71 @@
+import * as okta from '@okta/okta-sdk-nodejs';
+
+/**
+ * Subset of User information provided by Okta. See okta.User for further information on it's derived type.
+ * @see https://developer.okta.com/docs/reference/api/users/
+ */
+export type User = {
+  /** The internal Okta identifier. */
+  readonly id: string;
+  /**
+   * User login string.
+   */
+  readonly login: string;
+  /**
+   * User email adress.
+   */
+  readonly email: string;
+  /**
+   * Name of the user.
+   */
+  readonly name: string;
+  /**
+   * User status as a string.
+   */
+  readonly status: string;
+};
+
+// eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
+export const user = (oktaUser: okta.User) => ({
```

Comment:
Is there a better funtion name than `user` that describes this function? Something like `oktaUserToUser` or `simplifiyOktaUser`

Reply:
Yep, good question. 

Where possible, we tend to reserve the use of verbs for functions that have side effects.

What about `oktaUserAsUser` ?